cypress.config.ts

```diff
@@ -0,0 +1,26 @@
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    specPattern: 'cypress/integration/**/*.spec.{js,ts,jsx,tsx}',
+  },
+  video: true,
+  viewportHeight: 1920,
```

Comment:
The viewportHeight is set to 1920 and viewportWidth to 1080. Ensure these dimensions match the typical screen size you expect for testing. If testing for a standard desktop view, these values are fine, but consider adjusting if testing for other screen sizes.